bindings/python/scripts/dlite-validate

```diff
@@ -140,6 +140,22 @@ def main():
     for inst in insts:
         if inst.is_meta:
             check_dimensions(args.url, inst)
+            # Check that datamodels for properties that refer to
+            # a specific datamodel can be loaded.
+            for prop in inst.properties['properties']:
+                if prop.type == 'ref':
+                    if prop.ref == 'ref':
```

Comment:
This is a very specific case to check for. Don't you want to check that `prop.ref` is the URI for a valid datamodel?

Another question, would it make sense to allow `prop.ref` to be empty (just referring to a DLite instance (of any metadata)? There are very few requirements on a generic DLite instance, but one of them is that it must have a `meta` attribute referring to the metadata describing it. It would allow to build more flexible data structures that are not fully defined by the metadata (but who's instances are fully defined).

Reply:
This is evry specifc, but it is the case that occurs when the keyu ref is writte as `ref: uri` and not `$ref: uri`. It is a very easy mistake to make, that is why I added it.

Reply:
And yes, I check that the it is valid bu checking that it can be instantiated afterwards.